src/com/lftechnology/java/training/dipak/guess/GuessTheNumber.java

```diff
@@ -0,0 +1,84 @@
+package com.lftechnology.java.training.dipak.guess;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/*Guessing a number game application: 
+*application will generate a random no between 1 and 20. 
+*user guess the value by entering from the console
+*Check if the guess value matched with random number or not, if matched show the guessing value, random value and the no. of attempts. 
+ */
+
+public class GuessTheNumber{

```

Comment:
It is not the proper java doc comment style. See you are only using multi-line comment not java doc comment. Java doc comment should start with /**